packages/ui-components/src/styles/markdown.css

```diff
@@ -1,173 +1,178 @@
-main {
-  @apply flex
-    w-full
-    flex-col
-    gap-6;
-
-  hr {
-    @apply w-full
-      border-t
-      border-t-neutral-200
-      bg-white
-      dark:border-t-neutral-900
-      dark:bg-neutral-950;
-  }
-
-  h1 {
-    @apply text-3xl;
-  }
+@reference "tailwindcss";
```

Comment:
I see lots of css changes here, are they all needed?

Reply:
Yes. Every CSS file used by PostCSS _must_ contain a reference to the Tailwind import, and cannot contain a circular dependency, nor a `themes.scss` import within a layer

Reply:
It's all whitespace changes, not sure why GitHub is making seem drastic